packages/angular/src/builders/webpack-browser/schema.json

```diff
@@ -344,9 +344,14 @@
         "type": "string"
       },
       "default": []
+    },
+    "customBuilder": {
+      "description": "Override default Angular browser builder",
+      "type": "string",
+      "default": "@angular-devkit/build-angular:browser"
     }
   },
-  "additionalProperties": false,
+  "additionalProperties": true,
```

Comment:
Some custom builder such as custom webpack requires additional properties. That's the only real downside I can see from this change.